acceptance-tests/src/test/resources/features/site/search.feature

```diff
@@ -81,3 +81,17 @@ Feature: Basic search
         Given I navigate to the "home" page
         And I search for "_Attachment_Search_Term_"
         Then I should see 0 search results
+
+    Scenario: Search boosts are applied correctly to the different fields
+        Given I navigate to the "home" page
+        And I search for "WeightSearchTerm"
+            # Search results should go Series -> Publication -> Dataset
+            # within those groups the order should be Title -> Summary (-> Key Facts)
+        Then I should see search results:
```

Comment:
maybe: "I should see search results starting with"

Reply:
Good shout, will update